components/kyma-environment-broker/common/orchestration/dto.go

```diff
@@ -153,3 +153,15 @@ type StatusResponseList struct {
 type UpgradeResponse struct {
 	OrchestrationID string `json:"orchestrationID"`
 }
+
+type MaintenancePolicyEntry struct {
+	GlobalAccountID string         `json:"globalAccountID"`
+	Plan            string         `json:"plan"`
+	Days            []time.Weekday `json:"days"`
+	TimeBegin       time.Time      `json:"timeBegin"`
+	TimeEnd         time.Time      `json:"timeEnd"`
+}
+
+type MaintenancePolicy struct {
+	Rules []MaintenancePolicyEntry `json:"rules"`
```

Comment:
Just when giving a second thought, we should support a configurable default rule here:
```
type MaintenancePolicy struct {
	...
	Default MaintenancePolicyEntry `json:"rules"`
```

And the logic in manager should be that after iterating over the rules and no rules matched, we propagate any optional days/begin/end attributes if specified.